### ISSUE TITLE
Using >1 ElementalArea confuses content authors

### ISSUE THREAD

**Issue by phptek** [dnadesign/silverstripe-elemental#758]
For projects that have migrated to Elemental from other Blocks modules, CMS authors are used to seeing at-a-glance, which "Area" within a page, their block will be added to.

OOTB, I see no easy way to prepend a `HeadingField` or something similar before each `ElementalArea` - in fact, AFAICT, the standard Silverstripe forms API doesn't work as expected: If I try to use `$fields->insertBefore()` in an additional `DataExtension` subclass, the headings _are_ added, but in a _completely_ different spot in the CMS' UI.

I had to resort to this hackery to make it work (Unless of course I am missing something)

```

**Comment by brynwhyman** [comment]
Do you mean like this?

![image](https://user-images.githubusercontent.com/15868440/69828878-7f5bea00-1282-11ea-900c-a17562b08da8.png)

You can see the above implementation in the [Silverstripe demo](https://www.silverstripe.com/silverstripe-cms/demo/)

Codebase is here: [silverstripe/bambusa-installer](https://github.com/silverstripe/bambusa-installer/blob/master/app/src/Pages/BlocksPage.php)

**Comment by phptek** [comment]
Yeah, like that but make it OOTB functionality, part of the Elemental / SIlverstripe forms API.

Cheers!

**Comment by mfendeksilverstripe** [comment]
I recommend to avoid having multiple elemental areas on a page. See https://github.com/dnadesign/silverstripe-elemental/issues/759 for more details.

**Comment by michalkleiner** [comment]
I'd support making the heading part of the area, being optional.

**Comment by tractorcow** [comment]
My solution is to just use ElementalAreasExtension directly, and not the pages specific one, and then add the block headers manually.

same as the demo solution actually.

```
$fields->insertBefore('HeaderArea', HeaderField::create('HeaderAreaTitle', 'Page heading blocks'));
$fields->insertBefore('BodyArea', HeaderField::create('HeaderAreaTitle', 'Main content blocks'));
```
